snapshot/lib/snapshot/runner.rb

```diff
@@ -204,6 +208,20 @@ def erase_simulator(device_type)
       `xcrun simctl erase #{device_udid} &> /dev/null`
     end
 
+    def localize_simulator(device_type, language, locale)
+      device_udid = TestCommandGenerator.device_udid(device_type)
+      if device_udid
+        locale ||= language.sub("-", "_")
+        plist = {
+          AppleLocale: locale,
+          AppleLanguages: [language]
+        }
+        UI.message "Localizing #{device_type} (AppleLocale=#{locale} AppleLanguages=[#{language}])"
+        plist_path = "#{ENV['HOME']}/Library/Developer/CoreSimulator/Devices/#{device_udid}/data/Library/Preferences/.GlobalPreferences.plist"
+        File.write(plist_path, Plist::Emit.dump(plist))
```

Comment:
If the directory does not exist, that means that `erase_simulator`, which calls `xcrun simctl erase`, has failed. The directory structure is what is generated by `simctl erase`. I'm happy to put the `mkdir_p` if this code needs to be more stable then it is probably better to add some error handling and diagnostics to `erase_simulator`.